### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Aharonyn**
Updated app tests and increased coverage.

### CODE REVIEW

File: django_dicom/models/image.py [hunk 1]
```diff
@@ -46,13 +44,14 @@ class Image(DicomEntity):
     }
 
     class Meta:
+        ordering = ["-id"]
```
Comment (ZviBaratz) [inline django_dicom/models/image.py]: Added ordering to prevent DRF pagination warning.

File: django_dicom/models/image.py [hunk 2]
```diff
@@ -87,10 +86,7 @@ def get_data(self, as_json: bool = False) -> np.ndarray:
             Image's pixel array.
         """
 
-        data = self.read_file(header_only=False).pixel_array
-        if as_json:
-            return json.dumps({"data": data}, cls=NumpyEncoder)
-        return data
+        return self.read_file(header_only=False).pixel_array
```
Comment (ZviBaratz) [inline django_dicom/models/image.py]: Trying to return image data as JSON is ill-advised, therefore this option has been removed.

File: django_dicom/models/patient.py [hunk 3]
```diff
@@ -27,15 +26,6 @@ class Patient(DicomEntity):
...
 
-    if hasattr(settings, "SUBJECT_MODEL"):
-        subject = models.ForeignKey(
-            settings.SUBJECT_MODEL,
-            on_delete=models.PROTECT,
-            related_name="dicom_patient_set",
-            blank=True,
-            null=True,
-        )
-
```
Comment (ZviBaratz) [inline django_dicom/models/patient.py]: _django_dicom_ should not depend on external models, and therefore the Subject model association has been removed. This functionality is handled separately by _django_mri_.

File: django_dicom/models/series.py [hunk 4]
```diff
@@ -189,10 +189,11 @@ def get_data(self, as_json: bool = False) -> np.ndarray:
...
         images = self.image_set.order_by("number")
-        data = np.stack([image.get_data() for image in images], axis=-1)
-        if as_json:
-            return json.dumps({"data": data}, cls=NumpyEncoder)
-        return data
+        # data = np.stack([image.get_data() for image in images], axis=-1)
+        # if as_json:
+        #     return json.dumps({"data": data}, cls=NumpyEncoder)
+        # return data
+        return np.stack([image.get_data() for image in images], axis=-1)
```
Comment (ZviBaratz) [inline django_dicom/models/series.py]: Trying to return image data as JSON is ill-advised, therefore this option has been removed.

File: django_dicom/reader/parser.py [hunk 5]
```diff
@@ -131,9 +131,11 @@ def parse_date(self, element: DataElement) -> datetime.date:
...
                     f"Failed to parse {element.name} with value '{element.value}' into a valid date object"
                 )
-            # If the element is empty, return None
-            else:
-                return None
+            # If empty string, returns None
+        except TypeError:
+            # If the value is None, simply return None, else raise TypeError
+            if element.value:
+                raise
```
Comment (ZviBaratz) [inline django_dicom/reader/parser.py]: Simplified exception handling logic.

File: django_dicom/reader/parser.py [hunk 6]
```diff
@@ -159,12 +161,14 @@ def parse_time(self, element: DataElement) -> datetime.time:
...
                         f"Failed to parse {element.name} with value '{element.value}' into a valid time object!"
                     )
-            # If the value is empty, simply return None
-            else:
-                return None
+            # If the value is empty string, simply return None
+        except TypeError:
+            # If the value is empty, simply return None, else raise TypeError
+            if element.value:
+                raise
```
Comment (ZviBaratz) [inline django_dicom/reader/parser.py]: Simplified exception handling logic.

File: django_dicom/reader/parser.py [hunk 7]
```diff
@@ -337,7 +342,7 @@ def parse(self, element: DataElement):
             except KeyError:
                 return element.value
         except ValueError as e:
-            print(e.args)
+            # print(e.args)
```
Comment (ZviBaratz) [inline django_dicom/reader/parser.py]: What is the purpose of this comment?

Comment (Aharonyn) [reply django_dicom/reader/parser.py]: I commented it out in order to see a clean test output, should be there.

File: django_dicom/reader/parser.py (hunk 6 above)
Comment (ZviBaratz) [inline django_dicom/reader/parser.py]: Simplified exception handling logic.

**Pull request merged**